examples/websocket-proxy-server/websocket_proxy_server.bal

```diff
@@ -24,39 +24,47 @@ service<http:WebSocketService> SimpleProxyService bind { port: 9090 } {
         caller.attributes[ASSOCIATED_CONNECTION] = wsClientEp;
         //When creating client endpoint, if `readyOnClient` flag is set to false client endpoint does not start reading frames automatically.
         //So `ready` action of endpoint should be called separately when ready to accept messsages.
-        wsClientEp->ready()
-            but { error e => log:printError(
-                     "Error calling ready on client", err = e) };
+        var err = wsClientEp->ready();
+        if (err is error) {
+            log:printError("Error calling ready on client", err = err);
+        }
     }
 
     //This resource is triggered when a new text frame is received from a client.
     onText(endpoint caller, string text, boolean finalFrame) {
 
         endpoint http:WebSocketClient clientEp =
-                        getAssociatedClientEndpoint(caller);
-        clientEp->pushText(text, final = finalFrame)
-            but { error e => log:printError(
-                     "Error occurred when sending text message", err = e) };
+                    getAssociatedClientEndpoint(caller);
+        var err = clientEp->pushText(text, final = finalFrame);
+        if (err is error) {
+            log:printError("Error occurred when sending text message",
+                            err = err);
```

Comment:
It is necessary to have only upto 80 characters in Ballerina examples so that they will display properly in the website. Hence the file is formatted thus.